strapi/strapi
# Fix: Query Validation in Files' Admin API

### What does it do?

Fixes the query validation for the files' admin API.

It now only validates/sanitizes the request query, not the one generated with permissions/conditions. 

### Why is it needed?

See #18297

### How to test it?

`yarn test:api tests/api/core/upload/admin/file.test.api.js`

or see reproduction steps on #18519

### Related issue(s)/PR(s)

fix #18297
